.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: "\U0001F41BBug report"
 about: Create a report to help us improve
 title: ''
-labels: ''
+labels: Bug
```

Comment:
I suggest we do not label Bug automatically. Bugs have different SLA in terms of how soon it should be solved. We could have some numbers of false reports. We’d better manually label a bug after human investigation.

Reply:
@fengzhou-msft, do we have statistics about false report? If the percentage of false report is too high, we should find a way to resolve false report issue. Anyway, automation labeling is our goal.

Reply:
I don't think we have such statistics. @Juliehzl @MyronFanQiu @jiasli may have better sense about how often that happens since they've been in the team a little bit longer. 

One reason for false reports is that in the new issue [page](https://github.com/Azure/azure-cli/issues/new/choose), the option to `Open a blank issue` is so small comapred to the above three options. Users may not see it and would mostly select `Bug report` for cases like a question. 

Another frequent false report case is that it's just a usage issue such as passing wrong parameter, not a real bug.

B.T.W. we have issues coming from documentation, we may want to label those automatically as well.

Reply:
Bug label has two months resolve SLA which is one month shorter. I think we can create another label or just do not add Bug label.

Reply:
@fengzhou-msft , @yonzhan , I removed auto-labeling for bug template, please help review again.